src/ews_serialize.erl

```diff
@@ -143,6 +162,31 @@ encode_term(Term, #type{qname=Key, alias=A}, Tbl) when is_tuple(Term) ->
         false ->
             error({"expected #"++atom_to_list(A)++"{}", Term})
     end;
+encode_term(Term, #type{qname=Key, alias=A, attrs=[_|_]=PossAttrs}, Tbl)
+  when is_tuple(Term) ->
+    logger:debug("PossAttrs: ~p~n", [PossAttrs]),
+    [Name, Attrs|Values] = tuple_to_list(Term), %% TODO: Move this one clause up
```

Comment:
What does "move one clause up" entail?

If it's too big a change for this PR I think the TODO needs to be a bit more detailed, we can create an issue in this repo to track it.

Reply:
That was just a copy paste from the original clause. Hehe, and it is ancient! It is from 2014.

```
$ git log -p '-SMove this one clause up'
...
commit 2140430b8f88828649d42f72a93bd1e458b632f6
Author: Christoffer Vikström <cvi@campanja.com>
Date:   Tue May 6 23:54:14 2014 +0200

    Fix broken inheritance handling

```

Since I can't figure out if this comment is even relevant anymore I didn't want to touch it. I have seen it every time I worked on the code and wondered why it is there.

But it is fun to see some ancient comments from old colleagues.